src/components/sidebar/Sidebar.tsx

```diff
@@ -2,6 +2,17 @@ import React from 'react';
 import { Divider, Drawer, IconButton, List, ListItem, Toolbar, Tooltip } from '@mui/material';
 import { SideIcons } from './SideIcons';
 import { useNavigate } from 'react-router-dom';
+import axios from 'axios';
+import LogoutIcon from '@mui/icons-material/Logout';
+
+async function logoutFunction() {
```

Comment:
we know it's a function why add it to the name? it's redundant. Just call it `logout` or `handleLogout`
after you logout then what? Do you just stay on the page? navigate to the login page afterward.